src/components/Recidiviz/RecidivizSlider/RecidivizSlider.js

```diff
@@ -1,17 +1,31 @@
-import React from 'react';
+import React, { Component } from 'react';
 
-function RecidivizSlider() {
-  return (
-    <article className="bt b--black-10 black-70 ph3 ph5-ns pt2 pt3-ns">
-      <div className="mw9 center pa3 pa5-ns" id="first-chart">
-        <h1 className="f5 fw6 f2-ns lh-title measure mt0 pb5 text-our-gray">
-          If 100 people were released from prison today...(drag the slider)
-        </h1>
+import './RecidivizSlider.css';
+import addRecidivismSlider from '../../../lib/static/recidivismSlider';
```

Comment:
Is there a way to reference the `rootDir` in import statements, or to use paths implicitly relative from the root?

Reply:
Nice feature.